my-app/src/components/Firebase/fetchData.js

```diff
@@ -0,0 +1,70 @@
+const BASE_URL = 'https://petcare-app-264d9.firebaseio.com/';function getDoctors() {
```

Comment:
I believe that you have this `BASE_URL` already in environment variables, which you can take by `process.env....`

Reply:
I deleted fetchData file :)